Dev test

1237633 opened the pull request.

fifimova committed:
- добавила api doc для ShelterController

kurojiinn committed:
Создал сущность Report, Photo.
Исправил кнопки.

kurojiinn committed:
Исправления

1237633 committed:
Merge pull request #15 from 5ka-Inc/feature-6

Создал сущность Report, Photo.

1237633 committed:
Added scripts for new entities and small fixes in photo and report

1237633 committed:
Merge pull request #16 from 5ka-Inc/scripts-for-photo-and-reports

Added scripts for new entities and small fixes in photo and report

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/Photo.java:
```diff
@@ -0,0 +1,37 @@
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "id")
+    private UUID id;
```
мы же на Long перешли

fifimova replied:
раз везде Long, то смысл тут uuid делать

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/Report.java:
```diff
@@ -0,0 +1,56 @@
+public class Report {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
```
та же история

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/Report.java:
```diff
@@ -0,0 +1,56 @@
+    private UUID id;
+
+    @Column(name = "date")
+    private LocalDateTime date;
```
лучше Timestamp, там со временем точным

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/Report.java:
```diff
@@ -0,0 +1,56 @@
+    @Column(name = "is_report_valid")
+    private boolean isReportValid;
+
+    @OneToMany(mappedBy = "report")
```
связь OneToOne будет
на один отчет - одна фотка
отчёты каждый день буду с одной фоткой слаться

1237633 replied:
Вот насчет одной фотки не уверен. А если человек решит, что по одной ничего непонятно. Или болячки надо будет отфоткать, которых не одна

fifimova replied:
просто в тз было сказано в единственном числе, оттуда и плясала

kurojiinn committed:
добавления метода по созданию кнопок

kurojiinn committed:
исправления

kurojiinn committed:
Merge pull request #18 from 5ka-Inc/feature-4

добавления метода по созданию кнопок

fifimova committed:
Merge pull request #14 from 5ka-Inc/shelter-api

Shelter api

1237633 committed:
Added level1 menu. User is not crating anymore at 1st message. Reworked lvl0 menu. chatId variable is now being created out of cycle. Reworked update hub logic. No more Try/catch.

fifimova committed:
- пофиксила скрипты и импорты
-создала класс для универсальных клавиатур
- скопировала KeyboardBasicShelterOpsImpl.java из пр Антона, чтобы посмотреть как взаимодействует с утилой

fifimova committed:
fix

CAB4EHKO committed:
Написал документацию.

1237633 committed:
Merge pull request #20 from 5ka-Inc/fix-and-add-keyboard

fix-and-add-keyboard

fifimova committed:
fix

fifimova committed:
Merge pull request #22 from 5ka-Inc/fix-and-add-keyboard

финальные настройки клавиатур

fifimova committed:
Merge branch 'dev-test' into rework-menu-logic-test

fifimova committed:
Merge pull request #19 from 5ka-Inc/rework-menu-logic-test

Added level1 menu. User is not crating anymore at 1st message.

fifimova committed:
Merge branch 'dev-test' into documentation

fifimova committed:
Merge pull request #21 from 5ka-Inc/documentation

Написал документацию.

fifimova committed:
fix
